source/stylesheets/case-studies/easy-money/components/_image_showcase.scss

```diff
@@ -48,12 +48,42 @@
 
 .easy-ImageShowcase-main {
   z-index: 2;
+
+  width: 50%;
+
+  margin: 0 auto;
+
+  @include media('>=desktop') {
```

Comment:
Hmm didn't think it would generate more code and overrides. In these situations, non mobile first is actually ok. So maybe you could go back to what you had